app/workers/s3_raw.rb

```diff
@@ -23,7 +32,7 @@ def work
       loop do
         messages = queue.receive_messages(max_number_of_messages: 10)
         batched_messages += messages.to_a
-        break if messages.size == 0
+        break if messages.size == 0 || batched_messages.size > 100_000
```

Comment:
Still could use better smarts here but 100k goes through just fine and should keep us going for quite some time.

```
I, [2017-12-02T14:32:53.301041 #27605]  INFO -- Workers::S3Raw: Starting worker with 3600 second interval
I, [2017-12-02T14:32:53.301104 #27605]  INFO -- Workers::S3Raw: Starting batch from https://sqs.us-west-2.amazonaws.com/985752656544/ingest-measurements-to-s3-raw-prd to s3://safecastdata-us-west-2/ingest/prd/s3raw/2017-12-02/14/32
I, [2017-12-02T14:35:51.225409 #27605]  INFO -- Workers::S3Raw: Batching 100010 messages
I, [2017-12-02T14:35:57.190697 #27605]  INFO -- Workers::S3Raw: Wrote 39064543 bytes to s3://safecastdata-us-west-2/ingest/prd/s3raw/2017-12-02/14/32
```

Reply:
Hrm... well maybe not every time:

```
[ 5943.305392] Out of memory: Kill process 15501 (ruby) score 325 or sacrifice child
[ 5943.310262] Killed process 15501 (ruby) total-vm:1018704kB, anon-rss:666708kB, file-rss:0kB
```